wire-kotlin-generator/build.gradle

```diff
@@ -12,9 +12,7 @@ jar {
 
 dependencies {
   api project(':wire-schema')
-  // TODO(egorand): Ideally KotlinGenerator should generate platform-independent code, once runtime
-  // is 100% multiplatform find the right configuration to enforce this
-  implementation project(path: ':wire-runtime', configuration: 'jvmRuntimeElements')
+  implementation project(path: ':wire-runtime', configuration: 'runtimeElements')
```

Comment:
Is the configuration needed? It should just do the right thing with a normal project dep.

Reply:
Hmm, I remember it being unhappy with the normal dep. Lemme try again.

Reply:
We do this in SQLDelight for the JDBC sqlite driver depending on the multiplatform runtime